src/main/java/core/mate/academy/service/MachineProducer.java

```diff
@@ -1,11 +1,12 @@
-package core.mate.academy.service;
-
-import java.util.List;
-
-public interface MachineProducer {
-    /**
-     * In your implementations - create 2-3 models, add them to the list and return
-     * @return - the list of models
-     */
-    List<Object> get();
-}
+package core.mate.academy.service;
+
+import core.mate.academy.model.Machine;
+import java.util.List;
+
+public interface MachineProducer<T extends Machine> {
+    /**
+     * In your implementations - create 2-3 models, add them to the list and return
+     * @return - the list of models
+     */
+    List<? extends Machine> get();

```

Comment:
Unused type T.
You already have type, that extends Machine, it's your's parametrized T value.

`List<T> get();`

Reply:
not fixed